apps/src/codebridge/FileBrowser/index.tsx

```diff
@@ -288,22 +289,21 @@ const InnerFileBrowser = React.memo(
                       className={moduleStyles['button-kebab']}
                     >
                       <span className={moduleStyles['button-bar']}>
-                        <span onClick={() => moveFolderPrompt(f.id)}>
```

Comment:
Last I heard we wanted to keep move for accessibility reasons, did we change our mind on that?

Reply:
Oh, I can totally put it back. I was going off the jira task description!

Reply:
Ah sorry I didn't update that 😬

Reply:
No worries! Added them back.